internal/broker/testdata/TestIsAuthenticated/golden/error_when_can_not_cache_token/second_call

```diff
@@ -1,3 +1,3 @@
 access: retry
-data: '{"message": "could not update token: could not cache token"}'
+data: '{"message": "could not update cached info: could not cache info: could not create token directory: mkdir provider_cache_path: permission denied"}'
```

Comment:
I would write the complete path here for easier debugging by the sysadmin. "which path I can’t write to?"

Reply:
It does write the complete path. The problem here is that, as the cache path is based on the provider URI, it would create a variance in the golden files (since the provider port varies with each run), so I had to [redact it from the golden files](https://github.com/ubuntu/oidc-broker/blob/478c61c506fbaa45ab005ed73b462de5e4403a00/internal/broker/broker_test.go#L417)

Reply:
oh, perfect!